doc/source/cd/background.md

```diff
@@ -274,11 +274,8 @@ the reconstruction loss, e.g. if MSE is used:
 $\phi, \psi = \text{arg} \min_{\phi, \psi}\, \lVert \mathbf{X}-(\phi \circ \psi)\mathbf{X}\rVert^2$.
 However, untrained (randomly initialised) autoencoders can also be used.
 For an example, a `pytorch` autoencoder can be incorporated into a
-detector by packaging it as a callable function using
-`preprocess_drift` (from
-[alibi_detect.cd.pytorch.preprocess](../api/alibi_detect.cd.pytorch.preprocess.rst))
-and `partial` (see
-[here](https://docs.python.org/3/library/functools.html#functools.partial)):
+detector by packaging it as a callable function using {func}`~alibi_detect.cd.pytorch.preprocess.preprocess_drift`
+and {func}`~functools.partial`: 
```

Comment:
I assume these directives are for stylistic rendering? Perhaps we should add these rules to `CONTRIBUTING.md` (sort of separate from docstring conventions, which, for the time being, are on [alibi](https://github.com/SeldonIO/alibi/blob/master/CONTRIBUTING.md#docstrings)).

Reply:
Yep just stylistic, the old []() refs to api docs pages still work. These directives are just a bit more inline with what we use elsewhere, e.g. in docstrings and also the api pages themselves. If you're happy with these I'll add a note to `CONTRIBUTING.md`.